README.md

```diff
@@ -75,6 +75,8 @@ Usage of ./observatorium-api:
     	The percentage of mutex contention events that are reported in the mutex profile. (default 10)
   -debug.name string
     	A name to add as a prefix to log lines. (default "observatorium")
+  -experimental.traces.read.endpoint-template string
+    	A template replacing --read.traces.endpoint, such as http://jaeger-{tenant}-query:16686
```

Comment:
How does this flag relate to ` -traces.read.endpoint` what is the precedence if both are defined?

Reply:
I think would be nice to document the difference between the flags in the `Backends` section from the readme https://github.com/observatorium/api#backends